test/unit/compress-timestamp-plugin_test.js

```diff
@@ -146,7 +146,7 @@ describe('Timestamp compression plugin', function() {
             },
             {
                 name: 'The Target Value',
-                type: 'timestamp',
+                type: 'ISO8601',
                 value: '20071103T131805'
```

Comment:
Quoting from Wikipedia:
```
Alternatively, a format for duration based on combined date and time representations may be used by agreement between the communicating parties either in the basic format PYYYYMMDDThhmmss or in the extended format P[YYYY]-[MM]-[DD]T[hh]:[mm]:[ss].
```
Maybe the P is missing is that's what your refer to. This format was the compressed one agreed with the Thinking Things for its use inside the devices. The IoTAgent itself transforms it to the expanded representation using the time compression plugin.